addon/lib/shooter.js

```diff
@@ -286,6 +287,13 @@ const ShotContext = Class({
     },
     copyLink: function () {
       clipboard.set(this.shot.viewUrl, "text");
+      notifications.notify({
+        title: "Link Copied",
+        text: "Link to the shot detail copied to the clipboard.",
+        data: "",
```

Comment:
Removed optional attributes, changed text, added icon. Looks good with the icon.